.github/codecov.yml

```diff
@@ -8,4 +8,5 @@ ignore:
   - src/vm/errors
   - src/types/errors
   - hint_accountant
+  - src/hint_processor/cairo-1-hint-processor # TODO: Remove this line
```

Comment:
when we should remove this?

Reply:
When we add unit tests/integration tests for all hints